### ISSUE TITLE
Security-related question about a possible exposed credential

### ISSUE THREAD

**Issue by Emma-NYL** [miguelBhumanes/FinalMasterThesis#1]
Hello,

My name is Emma, I am conducting an academic study on possible credential exposure in public GitHub repositories.

While analyzing this repository, I found a string that may represent a credential. I'm including the code snippet below containing possible leakage. To avoid exposing sensitive information publicly, I marked the sensitive information.

Could you please help clarify whether the detected string is:
1. a real credential, or
2. a placeholder / example value?

Thank you for your time.

---

**Code snippet (sensitive values masked):**

```
from statsmodels.tsa.arima.model import ARIMA
from sklearn.metrics import mean_squared_error
import warnings
warnings.filterwarnings('ignore')

# Importing GDP and Inflation data from January 1999 to December 2022
fred = Fred(api_key="your****_here")
real_gdp_data = fred.get_series('GDPC1')
real_gdp_data = real_gdp_data['1999':'2022']
cpi_data = fred.get_series('CPIAUCSL')
cpi_data = cpi_data['1999':'2022']

# Computing the yearly rates of growth, which are the target variables

```

Thank you in advance for your time - I really appreciate it!

Sincerely,
Emma
